nadlogar/problems/models/eksponentna_funkcija.py

```diff
@@ -0,0 +1,166 @@
+import random
+
+import sympy
+
+from .meta import GeneratedDataIncorrect, Problem
+
+
+def naredi_eksponentno(do=3, cela_osnova=False, premik=0):
+    """
+    Funkcija vrne naključno eksponentno funkcijo, ki ustreza vpisanim pogojem.
+    """
+
+    x = sympy.symbols("x")
+    izbor = list(range(2, do + 1))
+    if not cela_osnova:
+        izbor += (
+            [sympy.Rational(x, 2) for x in range(1, 2 * (do)) if x != 2]
+            + [sympy.Rational(x, 3) for x in range(1, 3 * (do)) if x != 3]
+            + [sympy.Rational(x, 4) for x in range(1, 4 * (do)) if x != 4]
+            + [sympy.Rational(x, 5) for x in range(1, 5 * (do)) if x != 5]
+        )
+    osnova = random.choice(izbor)
+    return (osnova, premik, sympy.Add(sympy.Pow(osnova, x), premik, evaluate=False))
+
+
+class GrafEksponentne(Problem):
```

Comment:
Po mojem mora ta naloga počakati, dokler ne rešimo risanja grafov. Sicer se bo npr. pri izbiri tipa nalog vmes pojavljala neka čudna neprebavljena LaTeX koda in bodo ljudje mislili, da je nekaj narobe.